README.md

```diff
@@ -136,3 +150,15 @@ julia> rme_features[mismatched_unique, :LABEL_ID]
 # "11-310"
 # "11-311"
 ```
+
+## Adding Additional Data
+
+The following scripts add data to the initial .gpkg created by `1_create_canonical.jl`. The setup script (`1_create_canonical.jl`) must be run before the following scripts. These scripts make use of the `find_intersections` function from `common.jl`. Each script saves to the same file: `rrap_shared_lookup.gpkg`
+- `2_add_cots_priority.jl` - Adds the priority level for cots intervention for each reef. 
```

Comment:
Minor quibble here:

I would use colons here instead of dash for clarity, like so:

```
- `2_add_cots_priority.jl` : Adds the ...
```